plugin.audio.ampache/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.audio.ampache" version="1.2.0" name="Ampache" provider-name="lusum">
+    <requires>
+        <import addon="script.module.future" version="0.16.0"/>
```

Comment:
missing xbmc.python dependency

Reply:
Ok, i will add

Reply:
done

Reply:
i have added xbmc.python dependency 2.26.0, but now it doesn't work on kodi 19 ( works in kodi 18 btw )

Reply:
You have to have a different addon.xml for Leia and Matrix because of the change in Python versions (Python 2.x in Leia, Python 3.x in Matrix onward). So for the version you've submitted to the Leia branch, you need the 2.26.0 Python dependency.  You will have to submit separately to the Matrix repo, and that addon.xml has to have the 3.0.0  Python dependency.  It also has to have a difference version number (greater than what is in Leia). We suggest adding `+matrix.1` to the version for Matrix.

Reply:
In my repository i skip xbmc.python to mantain full compatibility, but if i need to add different addon.xml files i don't know how to do. I have to create different branches and merge them every time i made a commit or i simply add an addon.xml+matrix.1 file as you seems to suggest? Correct me if i'm wrong

Reply:
We require the Python dependency for the official repo.  For addons that are 100% the same code between Python2 and Python3, what I do is keep one branch on Github. When I'm ready to submit to the official repo, I submit to the Matrix branch first (with the addon.xml updated as needed), and then submit to the Leia branch after the Matrix one is approved.  For cases where there are code differences, I maintain two branches and manage them separately. We understand this is a bit of a pain, but this is the only time you'll have to do this between versions. Well, at least until we have to go to Python4.  '-)

Reply:
Ok, i will create two different branches and i will merge the diffs when i will submit on kodi repository, yes, a little bit a pain, but i think it is manegeable. Now i have only a couple of changes to do, i will wait some tips about my AmpachePlayer class ( not so necessary , so i can get rid of it if it become a problem ) and the check that the files played are from my plugin ( honestly i don't know how to do, the messages i receive from kodi don't include the  plugin source )